### DISCUSSION TITLE
TypeError: cols.map is not a function

### DISCUSSION THREAD

**Discussion by meo1** [quasarframework/quasar#14053] [General - Components / Directives / etc]
I get time to time this error message when I reload the page sometimes it works, sometimes it gives this strange error messages:

![image](https://user-images.githubusercontent.com/35503088/180962727-d56efe9b-6652-4b4b-ab1a-c07109190bc8.png)

![image](https://user-images.githubusercontent.com/35503088/180962844-11e01f48-a76b-4b6f-a922-7a6ba0a8330e.png)

![image](https://user-images.githubusercontent.com/35503088/180963224-2d19c041-fab0-4738-8d36-64793f0d8da9.png)

**Comment by MilosPaunovic** [comment]
Please provide a code reproduction of some kind.

**Comment by meo1** [comment]
It is very difficult to reproduce, if I can reproduce it, I may probably solve the problem.

But I can send you the whole code of the page where this error appears by hoping that may give you an idea. I send you .vue file but changed the extension to .txt, because Github does not accept .vue files to upload:
[ObjectsPage .txt](https://github.com/quasarframework/quasar/files/9188277/ObjectsPage.txt)

**Comment by MilosPaunovic** [comment]
> It is very difficult to reproduce, if I can reproduce it, I may probably solve the problem.

It's not valid for you to expect assistance from any open source contributor if you're not ready to invest some time in providing reproduction yourself.

**Reply by meo1** [reply]
OK, I will try. Thanks for quick reply.
